flink-ai-flow/ai_flow/test/plugin_interface/test_blob_manager.py

```diff
@@ -0,0 +1,50 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import unittest
+from typing import Text
+from ai_flow.plugin_interface.blob_manager_interface import BlobManagerFactory, BlobManager
+
+
+class MockBlockManager(BlobManager):
+    def __init__(self, config):
+        pass
+
+    def upload_blob(self, workflow_id: Text, prj_pkg_path: Text) -> Text:
+        return 'upload'
+
+    def download_blob(self, workflow_id, remote_path: Text, local_path: Text = None) -> Text:
+        return 'download'
+
+
+class TestBlobManager(unittest.TestCase):
+
+    def test_custom_blob_manager(self):
```

Comment:
Is the purpose of this test to test the blob_manager_factory? If so can we just rename the method to `test_blob_manager_factory()`, so it is clear that we are not testing the `MockBlockManager`.